### PR TITLE
Add section on upgrading CoreDNS

### PR DESCRIPTION
**Pull request by rajansandeep**

### CODE REVIEW

File: content/en/docs/tasks/administer-cluster/coredns.md [hunk 1]
```diff
@@ -69,6 +72,14 @@ kubeadm init --feature-gates=CoreDNS=false
 
 For versions 1.13 and later, follow the guide outlined [here](/docs/reference/setup-tools/kubeadm/kubeadm-init-phase#cmd-phase-addon).
 
+## Upgrading CoreDNS 
+
+CoreDNS is available in Kubernetes since v1.9.
```
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/coredns.md]: Could this use a `feature-state` shortcode?

**Comment by johnbelamaric** [comment]
content /lgtm

File: content/en/docs/tasks/administer-cluster/coredns.md (hunk 1 above)
Comment (neolit123) [reply content/en/docs/tasks/administer-cluster/coredns.md]: are you suggesting something like this @sftim ?
```
{{< feature-state for_k8s_version="v1.9" state="stable" >}}
```

Comment (sftim) [reply content/en/docs/tasks/administer-cluster/coredns.md]: That's an option, yeah. It's not always appropriate; do you think it's the right fit here @rajansandeep?

Comment (rajansandeep) [reply content/en/docs/tasks/administer-cluster/coredns.md]: @sftim I'm sorry, I don't understand the significance of the change suggested. What does it mean?

**Comment by fturib** [comment]
@sftim : can we go ahead here ? (I think there is a question in comment from @rajansandeep).
We need to have PR approved before Monday. Thanks!

File: content/en/docs/tasks/administer-cluster/coredns.md (hunk 1 above)
Comment (sftim) [reply content/en/docs/tasks/administer-cluster/coredns.md]: Ah, sorry @rajansandeep. Here's the style guide topic on [using `feature-state` shortcodes](https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#feature-state).

It's a snag; we could merge this as-is if we want and then tidy up in a later PR.

Comment (rajansandeep) [reply content/en/docs/tasks/administer-cluster/coredns.md]: @sftim Thank you for the link. I guess that it wouldn't make sense to add it here as CoreDNS has progressed from alpha in 1.9 to default in 1.13. Anyways, we can clean up later if it is required.

**Comment by rajansandeep** [comment]
> content /lgtm

@johnbelamaric I guess your lgtm was not caught by the bot.

File: content/en/docs/tasks/administer-cluster/coredns.md (hunk 1 above)
Comment (sftim) [reply content/en/docs/tasks/administer-cluster/coredns.md]: I'd still call out stable features, especially if they're newly stable in the last release or the one before it.
(There's no clear process for dropping those markers later though).

Comment (sftim) [reply content/en/docs/tasks/administer-cluster/coredns.md]: And, to be clear, I'm fine with this PR getting merged.

**Comment by rajansandeep** [comment]
@kubernetes/sig-docs-en-reviews for approval

**Comment by fturib** [comment]
@neolit123 : do you know how we can get this documentation approved and merged ? Thanks!

**Comment by neolit123** [comment]
@fturib if nobody is approving you should ping the #sig-docs slack channel.

/lgtm
/assign @jimangel 
^ ping relatively small change related to CoreDNS.

**Comment by DanyC97** [comment]
@rajansandeep @fturib thank you for the PR

@sftim thank you too for reviewing it, i agree will be nice to have the shortcode, let's chase it on a different PR ourselves, maybe when we're going to align the style consistency on the concept section

/lgtm

@cody-clark pls help review it

**Pull request merged**